toolkit/tools/internal/toolchain/downloader.go

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Rename this so it doesn't collide with the existing command line `downlaoder` tool?